extensions/wikia/AdEngine/js/provider/gpt/sourcePointTag.js

```diff
@@ -38,44 +38,27 @@ define('ext.wikia.adEngine.provider.gpt.sourcePointTag', [
 			window.dispatchEvent(spReadyEvent);
 		});
 
-		doc.addEventListener('sp.blocking', (function () {
-			log(['sp.blocking'], 'debug', logGroup);
-			doc.body.classList.add('source-point');
-			blocking = true;
-		}).bind(this));
+		doc.body.classList.add('source-point');
 
-		doc.addEventListener('sp.not_blocking', (function () {
-			log(['sp.not_blocking'], 'debug', logGroup);
-		}).bind(this));
+		delete window.googletag;
```

Comment:
Add a comment here explaining why we have to do that. I believe it's connected to the note in the ticket about AdBlock Plus.